apps/web/app/(base-org)/getstarted/page.tsx

```diff
@@ -0,0 +1,51 @@
+import classNames from 'classnames';
+import AnalyticsProvider from '../../../contexts/Analytics';
+import Hero from '../../../src/components/GetStarted/Hero';
+import BuildersMostWanted from '../../../src/components/GetStarted/BuildersMostWanted';
+import Funding from '../../../src/components/GetStarted/Funding';
+import GetNoticed from '../../../src/components/GetStarted/GetNoticed';
+import StartBuilding from '../../../src/components/GetStarted/StartBuilding';
+import BuildWithUsFooter from '../../../src/components/GetStarted/BuildWithUsFooter';
+
+export const sectionContainerClasses = classNames(
+  'max-w-[1440px]',
+  'mx-12 sm:mx-16 lg:mx-24',
+  'mb-12 sm:mb-16 lg:mb-20',
+  'bg-black',
+);
+export const titleClasses = classNames(
```

Comment:
Hm, I wouldn't expect to import CSS classes from a page component like this. If U intend to reuse styles then I would expect the style to get packaged together with a component. In a React codebase I'd feel really surprised if modifying a class string effected a bunch of other components.

Reply:
moved all shared classes into a file in the component directory